modules/cnf-topology-aware-lifecycle-operator-preparing-for-updates.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+// Epic CNF-2600 (CNF-2133) (4.10), Story TELCODOCS-285
+// * scalability_and_performance/ztp-deploying-disconnected.adoc
+
+:_content-type: PROCEDURE
+[id="talo-platform-prepare-end-to-end_{context}"]
+= End-to-end procedures for updating clusters in a disconnected environment
```

Comment:
Add a short description something like the following

If you have deployed your spoke clusters with distributed unit(DU) profiles using the GitOps ZTP with Topology Aware Lifecycle Operator pipeline described in "Deploying distributed units at scale in a disconnected environment", this procedure tells you how to upgrade your spoke clusters and Operators.